### ISSUE TITLE
Support Django 4

### ISSUE THREAD

**Issue by jhrdt** [jonasundderwolf/django-admin-locking#6]
Hi,

I moved a Django 3.2 project to 4.0.

django-admin-locking imports some modules that are removed in Django 4.0.

I detected the following issues:

* Import of *ugettext* in admin.py
* Import of *url* (from django.conf.urls) in admin.py

See also:

* https://docs.djangoproject.com/en/4.0/releases/4.0/#features-removed-in-4-0

## Error Message

```
$ ./manage.py runserver

Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/core/management/commands/runserver.py", line 115, in inner_run
    autoreload.raise_last_exception()
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/utils/autoreload.py", line 87, in raise_last_exception
    raise _exception[1]
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 381, in execute
    autoreload.check_errors(django.setup)()
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/apps/config.py", line 223, in create
    import_module(entry)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/admin_locking/__init__.py", line 1, in <module>
    from .admin import AdminLockingMixin
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/admin_locking/admin.py", line 3, in <module>
    from django.conf.urls import url
ImportError: cannot import name 'url' from 'django.conf.urls' (/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/conf/urls/__init__.py)
```

```
$ ./manage.py runserver

Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/django/core/management/commands/runserver.py", line 115, in inner_run
    autoreload.raise_last_exception()
  File "/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/django/utils/autoreload.py", line 87, in raise_last_exception
    raise _exception[1]
  File "/home/developer/Code/demo/venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 381, in execute
    autoreload.check_errors(django.setup)()
  File "/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/django/apps/config.py", line 223, in create
    import_module(entry)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/admin_locking/__init__.py", line 1, in <module>
    from .admin import AdminLockingMixin
  File "/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/admin_locking/admin.py", line 3, in <module>
    from django.conf.urls import url
ImportError: cannot import name 'url' from 'django.conf.urls' (/home/developer/Code/dj-literaturverwaltung/venv/lib/python3.8/site-packages/django/conf/urls/__init__.py)
```

**Comment by michjnich** [comment]
There's been a fix for a year. Is it going to get uploaded as a new version on pypi?

**Comment by anrie** [comment]
A new version with the fixes was released as v0.10.0 (https://pypi.org/project/django-admin-locking/0.10.0/)
